fe/fe-core/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -324,17 +324,22 @@ public void getTables(Analyzer analyzer, Map<Long, Table> tableMap, Set<String>
     }
 
     @Override
-    public void getTableRefs(List<TableRef> tblRefs, Set<String> parentViewNameSet) {
-        getWithClauseTableRefs(tblRefs, parentViewNameSet);
+    public void getTableRefs(Analyzer analyzer, List<TableRef> tblRefs, Set<String> parentViewNameSet) {
+        getWithClauseTableRefs(analyzer, tblRefs, parentViewNameSet);
         for (TableRef tblRef : fromClause_) {
-            if (tblRef instanceof InlineViewRef) {
-                QueryStmt inlineStmt = ((InlineViewRef) tblRef).getViewStmt();
-                inlineStmt.getTableRefs(tblRefs, parentViewNameSet);
-            } else {
-                if (isViewTableRef(tblRef.getName().toString(), parentViewNameSet)) {
-                    continue;
+            try {
+                tblRef = analyzer.resolveTableRef(tblRef);
```

Comment:
Better using a local variable to save the return value like:
```
TableRef tmpTblRef = analyzer.resolveTableRef(tblRef);
```

Reply:
Ok